README.md

```diff
@@ -220,301 +57,72 @@ module.exports = {
-  -g, --group [name]                 [dashboard] group recorded tests together under a single run
-  -t, --tag <name>                   [dashboard] add tags to dashboard for test run
-  -h, --help                         display help for command
+4. **(Optional)** Create a custom config file. @agoric/synpress aleardy has some configurations set up in this [file](https://github.com/agoric-labs/synpress/blob/master/synpress.config.js). To override this and add your custom config, you can create your own config file `synpress.config.js` in `/project_dir/tests/e2e`
+```js
+const baseConfig = require('@agoric/synpress/synpress.config');
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  ...baseConfig,
+  e2e: {
+    ...baseConfig.e2e,
+    baseUrl: 'http://localhost:5173',
+  },
+});
 ```
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+use this config by passing the `--configFile` flag to synpress
+```json
+{
+  ...
+  "scripts": {
+    ...
+    "test:e2e": "EXTENSION=keplr synpress run --configFile=test/e2e/synpress.config.js"
+  }
+}
 
-```text
-Usage: synpress open [options]
-
-launch test runner UI
-
-Options:
-  -cf, --configFile <path>  specify a path to *.js file where configuration values are set
-  -h, --help                display help for command
 ```
 
-## 👨‍⚕️ Known problems with MetaMask
 
-If your MetaMask is stuck on the loading screen, check what's happening under the hood in the console. You can find vital information about why it's stuck on this step.
+5. You're done! 🎉
 
-#### ⭐ Sentry.io HTTP error 499 (Request has been forbidden by antivirus)
+</br>
 
-- Kaspersky antivirus sometimes blocks encrypted requests to Sentry.io. You can disable
-  this feature in Kaspersky advanced settings by toggling on `"Do not scan encrypted connections"`.
+For an example project, you can take a look at how we've set up tests in this [repository](https://github.com/agoric-labs/synpress/tree/dev/tests/e2e) 
 
 ## 📃 Environmental variables
 
 | Variable                | Description                                                                                                                                                                                                 |
 | ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `EXTENSION` (Required)          | Picks which extension to use during tests. `keplr` and `metamask` are the only possible values                                                                                                                              |
 | `SECRET_WORDS`          | Space separated words for the test wallet recovery phrase (mnemonic; 12 words)                                                                                                                              |
```

Comment:
I think it should be 1`2/24 words`?

Reply:
we actually only support 24 words at the moment. updated it accoridingly